raiden/api/rest.py

```diff
@@ -131,7 +131,13 @@ def normalize_events_list(old_list):
     new_list = []
     for _event in old_list:
         new_event = dict(_event)
-        new_event['event_type'] = new_event.pop('_event_type')
```

Comment:
I think it would be nicer to add a `decode()` here and add an `encode()` here: https://github.com/raiden-network/raiden/blob/master/raiden/api/python.py#L697